server/db/models/order.js

```diff
@@ -0,0 +1,87 @@
+const Sequelize = require('sequelize')
+const db = require('../db');
+const {OrderProduct} = ('../db/models');
+
+//foreign id: userId
+const Order = db.define('order', {
+  status: {
+    type: Sequelize.ENUM,
+    values: ['created', 'processing', 'shipped', 'cancelled', 'delivered'],
+    defaultValue: 'created'
+  },
+  firstNameShipping: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  lastNameShipping: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  firstNameBilling: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  lastNameBilling: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  shippingAddress : {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  billingAddress: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  ccNumber: {
+    type: Sequelize.INTEGER,
+    allowNull: false,
+  },
+  ccExpiration: {
+    type: Sequelize.INTEGER,
+  },
+  ccSecurity: {
+    type: Sequelize.INTEGER,
+  },
+  dateProcessed: {
+    type: Sequelize.DATE,
+  },
+  dateShipped: {
+    type: Sequelize.DATE,
+  },
+  dateDelivered: {
+    type: Sequelize.DATE,
+  },
+  totalPrice: {
```

Comment:
This might be a good instance to use a getter method. This way you don't have to update the DB every time something is added to the order